valhalla/mjolnir/adminbuilder.h

```diff
@@ -1,6 +1,6 @@
 #pragma once
 
-#include <boost/property_tree/ptree.hpp>
+#include <boost/property_tree/ptree_fwd.hpp>
```

Comment:
Generally speaking, there is no point in this replace as the only thing in this header could only be used with `boost::property_tree::ptree` so this header will always be included together with `#include <boost/property_tree/ptree.hpp>`.

Same comment to most of other `ptree.hpp` -> `ptree_fwd.hpp` replacements